test/kubemark/cloud-provider-config.sh

```diff
@@ -15,3 +15,6 @@
 # limitations under the License.
 
 CLOUD_PROVIDER="${CLOUD_PROVIDER:-gce}"
+CONTAINER_REGISTRY="${CONTAINER_REGISTRY:-gcr.io}"
+PROJECT="${PROJECT:-}"
```

Comment:
I think we should have a default value for ```PROJECT```. If ```PROJECT``` gets passed into make as ```''```, then the image will be built as ```gcr.io//kubemark```.